server/monitor.go

```diff
@@ -2259,11 +2259,12 @@ type JSInfo struct {
 	Disabled bool            `json:"disabled,omitempty"`
 	Config   JetStreamConfig `json:"config,omitempty"`
 	JetStreamStats
-	StreamCnt    int          `json:"total_streams,omitempty"`
-	ConsumerCnt  int          `json:"total_consumers,omitempty"`
-	MessageCnt   uint64       `json:"total_messages,omitempty"`
-	MessageBytes uint64       `json:"total_message_bytes,omitempty"`
-	Meta         *ClusterInfo `json:"meta_cluster,omitempty"`
+	APICalls  int64        `json:"current_api_calls"`
```

Comment:
what does this indicate exactly? Looks like in-flight api calls right?

Reply:
Yes for the ones we spin up Go routines for, but thinking about it we should account for them regardless of inline or not. Will also add this logic to getNextMsg as well in separate PR.